### PR TITLE
Update README with Detailed WebUI Service Launch Instructions

### PR DESCRIPTION
**Pull request by cklogic**
### What problem does this PR solve?

Improve README by detailing Launch Service from Source section

This commit enhances the README document by adding comprehensive steps for running the WebUI service in the 'Launch Service from Source' section. It aims to provide clearer guidance for users attempting to start the service from the source code, making the setup process more accessible and understandable.

Key changes include:
- Detailed instructions for setting up and running the WebUI service.
- Necessary prerequisites for launching the service from source.

This update ensures that users have all the information they need to successfully launch the service, improving the overall usability of our project.

### Type of change

- [ ] Bug Fix (non-breaking change which fixes an issue)
- [ ] New Feature (non-breaking change which adds functionality)
- [x] Documentation Update
- [ ] Refactoring
- [ ] Performance Improvement
- [ ] Other (please describe):

### CODE REVIEW

**Review by KevinHuSh — APPROVED** [review]

File: README_zh.md [hunk 1]
```diff
@@ -247,7 +247,28 @@ $ docker compose -f docker-compose-base.yml up -d
 $ chmod +x ./entrypoint.sh
 $ bash ./entrypoint.sh
 ```
+7. 启动WebUI服务
+```bash
+$ cd web
+$ npm install --registry=https://registry.npmmirror.com --force
+$ vim .umirc.ts
+# 修改proxy.target为127.0.0.1:9380
```
Comment (writinwaters) [inline README_zh.md:255]:
```suggestion
# 修改 proxy.target 为 127.0.0.1:9380
```

File: README_zh.md [hunk 2]
```diff
@@ -247,7 +247,28 @@ $ docker compose -f docker-compose-base.yml up -d
 $ chmod +x ./entrypoint.sh
 $ bash ./entrypoint.sh
 ```
+7. 启动WebUI服务
```
Comment (writinwaters) [inline README_zh.md:250]:
```suggestion
7. 启动 WebUI 服务
```

File: README.md [hunk 3]
```diff
@@ -247,6 +247,29 @@ $ chmod +x ./entrypoint.sh
...
+7. Start the WebUI service
+```bash
+$ cd web
+$ npm install --registry=https://registry.npmmirror.com --force
+$ vim .umirc.ts
+# Modify proxy.target to 127.0.0.1:9380
+$ npm run dev 
+```
+
+8. Deploy the WebUI service
```
Comment (writinwaters) [inline README.md:259]:
```suggestion
8. Deploy the WebUI service:

```

File: README_zh.md [hunk 4]
```diff
@@ -247,7 +247,28 @@ $ docker compose -f docker-compose-base.yml up -d
...
+```bash
+$ cd web
+$ npm install --registry=https://registry.npmmirror.com --force
+$ vim .umirc.ts
+# 修改proxy.target为127.0.0.1:9380
+$ npm run dev 
+```
 
+8. 部署WebUI服务
```
Comment (writinwaters) [inline README_zh.md:259]:
```suggestion
8. 部署 WebUI 服务：
```

**Review by writinwaters — CHANGES_REQUESTED** [review]
LGTM, with some nitpicking changes.

**Review by writinwaters — APPROVED** [review]

**Pull request merged**